src/reports/package/footer-text-for-service.tsx

```diff
@@ -2,19 +2,18 @@
 // Licensed under the MIT License.
 import * as React from 'react';
 
-import { ReactFCWithDisplayName } from 'common/react/named-fc';
-import { ReportFooterProps } from 'reports/components/report-sections/report-footer';
+import { FooterText } from 'reports/components/report-sections/footer-text';
 import { ToolLink } from 'reports/components/report-sections/tool-link';
 
-export const PackageReportFooter = (serviceName: string, reportFooter: React.FC<ReportFooterProps>) => {
+export const FooterTextForService = (serviceName: string) => {
 
-    const footerText = ({ axeCoreVersion, browserSpec }) =>
+    const footerText = ({ environmentInfo: { axeCoreVersion, browserSpec } }) =>
         <>
             This automated checks result was generated using the {serviceName}{' '}
             that helps find some of the most common accessibility issues. The scan was
             performed using axe-core {axeCoreVersion} and {browserSpec}. For a complete
             WCAG 2.1 compliance assessment please visit{' '}<ToolLink />.
         </>;
 
-    return ((props: ReportFooterProps) => reportFooter({ ...props, footerText })) as ReactFCWithDisplayName<ReportFooterProps>;
+    return footerText as typeof FooterText;
```

Comment:
I think this type should be couple to the `ReportSectionFactory` (which can be considered abstract in this context) instead of the `FooterText` (which can be considered an implementation in this context).